apache/superset
# chore: Select component refactoring - DndColumnSelectControl - Iteration 5

### SUMMARY
It replaces the react-select Select component with the Antdesign one in the DndColumnSelectControl/ColumnSelectPopover component.

### BEFORE

https://user-images.githubusercontent.com/60598000/130762141-59214722-959b-4cfd-bf3d-a2e7f5ae11f7.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/130762213-89318263-31c2-43d6-9f87-b267112e292c.mp4

### TESTING INSTRUCTIONS
1. Open a Bar Chart in Explore
2. Add new Series
3. Observe the dropdowns in Saved and Simple tabs

### ADDITIONAL INFORMATION
